content/working_with/console/widgets/blueprints.md

```diff
@@ -83,4 +97,8 @@ See Settings section for details on how to turn on/off this feature.
 
 * `Refresh time interval` - The time interval in which the widget’s data will be refreshed, in seconds. Default: 10 seconds
 * `Enable click to drill down` - This option enables redirecting to the blueprint’s drill-down page upon clicking on a specific blueprint. Default: True
-* `Display style` - Can be either Catalog or table. The deployments status column is only available in list mode.  Default: table
+* `Display style` - Defines how the blueprints list should be displayed. Can be either Catalog or Table. The deployments status column is only available in list mode.  Default: Table
```

Comment:
What does "The deployments status column is only available in list mode." mean?

Deployment count is visible in both - Catalog and Table view.
Blueprint status is visible in both - Catalog and Table view.

So I think it should be removed:

```suggestion
* `Display style` - Defines how the blueprints list should be displayed. Can be either Catalog or Table. Default: Table
```